intellst-api/src/Controller/AuthController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use App\Entity\User;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+
+class AuthController extends AbstractController
+{
+    /**
+     * @Route("/register", methods={"POST"})
+     */
+    public function register(Request $request, UserPasswordEncoderInterface $encoder) : string
+    {
+        $em = $this->getDoctrine()->getManager();
```

Comment:
please move to a separate service, which will be injected & called here in controller

Reply:
I don't understund how to do that

Reply:
i already did that :)